### PR TITLE
Release notes for April 21, 2020 Release

### PR DESCRIPTION
**Pull request by agau4779**

### CODE REVIEW

File: firestore-bigquery-export/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## Version 0.1.4
+
+fixed - Converting circular structure to JSON error (fixes issue #236).
```
Comment (rachelsaunders) [inline firestore-bigquery-export/CHANGELOG.md:3]:
```suggestion
fixed - Converted circular structure to JSON error. (Issue #236)
```

File: firestore-bigquery-export/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+fixed - Fixed bug where modules were not sharing the same Firestore
+DocumentReference. (fixes issue #265)
```
Comment (rachelsaunders) [inline firestore-bigquery-export/CHANGELOG.md:5]:
```suggestion
fixed - Fixed bug where modules were not sharing the same Cloud Firestore
DocumentReference. (Issue #265)
```

File: firestore-bigquery-export/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
...
+fixed - Updated @firebaseextensions/firestore-bigquery-change-tracker dependency (fixes issues #250 and #196).
```
Comment (rachelsaunders) [inline firestore-bigquery-export/CHANGELOG.md:6]:
```suggestion
fixed - Updated @firebaseextensions/firestore-bigquery-change-tracker dependency. (Issues #250 and #196)
```

Comment (rachelsaunders) [reply firestore-bigquery-export/CHANGELOG.md:6]: I wonder if this is a "changed" or a "fixed" update?

File: firestore-bigquery-export/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
...
+
 ## Version 0.1.3
 
 feature - Interpret data more easily with column descriptions in the exported BigQuery data (#138).
```
Comment (rachelsaunders) [inline firestore-bigquery-export/CHANGELOG.md:10]:
```suggestion
feature - Interpret data more easily with column descriptions in the exported BigQuery data. (PR #138)
```

File: storage-resize-images/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,7 @@
+## Version 0.1.6
+
+fixed - Switched ImageMagick for Sharp library to support webp format. (issue #199)
```
Comment (rachelsaunders) [inline storage-resize-images/CHANGELOG.md:3]:
```suggestion
fixed - Switched ImageMagick for Sharp library to support webp format. (Issue #199)
```

File: storage-resize-images/CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,7 @@
...
+
 ## Version 0.1.5
 
 fixed - The original, uploaded image's MIME type must now always be specified in the `Content-Type` header. (issue #175)
```
Comment (rachelsaunders) [inline storage-resize-images/CHANGELOG.md:7]:
```suggestion
fixed - The original, uploaded image's MIME type must now always be specified in the `Content-Type` header. (Issue #175)
```

File: firestore-bigquery-export/CHANGELOG.md (hunk 3 above)
Comment (agau4779) [reply firestore-bigquery-export/CHANGELOG.md:6]: Hmm - I'd leave it as fixed, because the fundamental behavior of the extension hasn't changed

**Pull request merged**